lib/Launcher.js

```diff
@@ -22,7 +22,7 @@ const Browser = require('./Browser');
 const readline = require('readline');
 const crypto = require('crypto');
 
-const CHROME_PROFILE_PATH = path.resolve(__dirname, '..', '.dev_profile');
+const CHROME_PROFILE_PATH = path.join(os.tmpdir(), 'puppeteer_dev_profile');
```

Comment:
It also turned out there's a `fs.mkdtemp` - let's use it in favor of current folder name generation.